docs/yaml/elements/election.yaml

```diff
@@ -1,10 +1,10 @@
 name: Election
 tags:
 - _name: Date
-  description: Specifies when the election is being held. The Date is considered to
+  description: Specifies when the election is being held. The `Date` is considered to
     be in the timezone local to the state holding the election.
   error: If the field is not present or invalid, the implementation is required to
-    ignore the election element containing it.
+    ignore the `Election` element containing it.
```

Comment:
For this one, you can delete the `error` key-value entirely.  What you have is what the default message will be.  (Going backwards on this and automatically performing this "DRY-ing" up was one of the things I didn't have time to get to.)